core/src/main/java/org/apache/carbondata/core/metadata/schema/table/CarbonTable.java

```diff
@@ -482,7 +482,7 @@ public String getTableUniqueName() {
    * @return
    */
   public boolean isLocalDictionaryEnabled() {
-    return isLocalDictionaryEnabled;
```

Comment:
why it is always false

Reply:
Currently Query part is not handled, so query will fail if local dictionary is enabled. I am working on the same